e2e/rnTestUtilities.js

```diff
@@ -80,19 +80,125 @@ const assertAutotrackHierarchy = async (
   }
 };
 
+const assertAndroidNavigationEvent = async (expectedPath, expectedType) => {
+  let props = {
+    path: {
+      string: expectedPath,
+    },
+  };
+  if (expectedType) {
+    props = {
+      ...props,
+      type: {
+        string: expectedType,
+      },
+    };
+  }
+
+  return assertAndroidEvent({
+    envId: '2084764307',
+    event: {
+      custom: {
+        name: 'reactNavigationScreenview',
+        properties: props,
+      },
+    },
+  });
+};
+
+const assertIosNavigationEvent = async (expectedPath, expectedType) => {
+  let props = ['path', expectedPath];
+  if (expectedType) {
+    props = [...props, 'type', expectedType];
+  }
+  return assertIosPixel({
+    t: 'reactNavigationScreenview',
+    k: props,
+  });
+};
+
+const assertNavigationEvent = async (expectedPath, expectedType) => {
+  if (device.getPlatform() === 'android') {
+    return assertAndroidNavigationEvent(expectedPath, expectedType);
+  } else if (device.getPlatform() === 'ios') {
+    return assertIosNavigationEvent(expectedPath, expectedType);
+  } else {
+    throw new Error(`Unknown device type: ${device.getPlatform()}`);
+  }
+};
+
 waitForEventsToFlush = async () => {
-  // Heap for iOS and Android flushes events every 15 seconds. Wait 16 seconds to ensure that
-  // all events are flushed to redis before asserting.
-  // :TODO:(jmtaber129): Make this wait shorter if/when we expose setting the
-  // flush frequency to the RN bridge.
```

Comment:
Given that we're removing this TODO, should we be lowering the delay to ~2 seconds?

Also nit re the replacement comment – not sure if there's a second half of that sentence.

Reply:
Discussed in person – this is probably moot now.